xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -2493,9 +2514,10 @@ bool CVppPostproc::Filter(CVaapiProcessedPicture &outPic)
   }
   outPic.DVDPic.SetParams(it->DVDPic);
 
+  const bool skipDeinterlacer = ((m_vppMethod == VS_INTERLACEMETHOD_NONE) || (outPic.DVDPic.iFlags & DVD_CODEC_CTRL_SKIPDEINT));
```

Comment:
why did you change this? you broke behaviour. SKIPDEINT skips the 2nd cycle but not entire deinterlacing

Reply:
Unintentional, will take a look

Reply:
OK should be fine the way it was even with handling `VS_INTERLACEMETHOD_NONE`. I probably misread something.